pom.xml

```diff
@@ -44,16 +44,18 @@
         <maven.compiler.target>1.7</maven.compiler.target>
         <maven.compiler.source>1.7</maven.compiler.source>
 
-        <wildfly.version>11.0.0.Final</wildfly.version>
-        <wildfly.build-tools.version>1.2.2.Final</wildfly.build-tools.version>
+        <wildfly.version>13.0.0.Final</wildfly.version>
+        <wildfly.build-tools.version>1.2.10.Final</wildfly.build-tools.version>
         <eap.version>7.1.3.GA-redhat-2</eap.version>
```

Comment:
This needs to be changed to corresponding EAP CD build. That's 7.2.0.CD13-redhat-3. Also need to check that RH-SSO works as expected.

Reply:
Ok, will look at it